### ISSUE TITLE
Why set no-cache? it load slows

### ISSUE THREAD

**Issue by ykt** [Pantau-RAPBD/pantau-rapbd.github.io#8]
Curios, why set `no-cache` on the meta tag? is it due to data.json ? Even cloning repo is slow.

**Comment by fikriauliya** [comment]
It's due to *.js and *.css. As simple html, the project doesn't have digest mechanism to update assets cache (such as Rails assets pipeline). Thus, we manually add  ?v=xx appendix to refresh the cache. Adding gulp or grunt here wasn't an option since it would add learning curve to setup the project. 

Slow repo cloning is totally separate issue. Probably just due to the Internet connection? The project is quite small actually.

Slow page load is due to Firebase connection I suspect. If I had time, I would choose to reimplement without firebase. Anyway, this project is my first firebase experience :smile:

**Comment by ykt** [comment]
I think it was slow, due to `*.json` or `*.csv` (~10 MB in total) data that is being cloned . I am not familiar `firebase`, but I think I can help to convert this project easily to nodeJs and run on Heroku, with Gruntfile. If you dont mind. :)

**Comment by fikriauliya** [comment]
I chose to host it in GitHub for both simplicity and easy-scalability, with zero cost :D. The project was created to be very simple to encourage more participation. I did consider using Gulp to minify/add digest to the assets, but decided not to do so as it add the learning curve to people (need to understand NPM at least).

Of course there are some drawbacks of this client-only model, e.g. the page load is slower as data.json is loaded in its entirety on page load (but it also makes search very fast on PC). But it also freed us from managing the scalability issues/server down/database bottleneck/etc.

Based on Google Analytics data, now the web view has decreased quite a lot (10-30 realtime views vs 100+ realtime views when it was first launched & many news portals put links to the website). If I the hype were still the same as before, it would be justified to make architectural change. However, as for now, I think doing migration wouldn't worth the effort.
